### PR TITLE
Feature/25 add skip option to location state

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
@hodgestar ready for help :)

File: lib/location.js [hunk 1]
```diff
@@ -97,6 +97,7 @@ var LocationState = State.extend(function (self, name, opts) {
         characters_per_page:160,
         next_text:"Next",
         previous_text:"Previous",
+        skip_text: "Skip",
```
Comment (hodgestar) [inline lib/location.js]: We need to maintain compatibility for existing applications so `skip_text` should probably default to `null` and then we should only add the skip handler and display `s. Skip` if `skip_text` is not null.

**Comment by hodgestar** [comment]
It looks like the two tests that are failing are just failing because with 's. Skip' added, the way the results are paginated changes. Following the making-`skip_text`-null-by-default suggestion should fix all the existing tests and then we can add new tests for when the skipping option is displayed.

**Comment by hodgestar** [comment]
Looking really good otherwise.

File: lib/location.js (hunk 1 above)
Comment (codiebeulaine) [inline lib/location.js]: display s. Skip if skip_test is null or is not null? Where would one make it not null?

Comment (hodgestar) [inline lib/location.js]: We'd set `skip_text` to not null when we instantiate the location state (e.g. `new LocationState("my_state", {skip: "Skip"})`).

**Comment by codiebeulaine** [comment]
@hodgestar changed some stuff but getting new errors from the example folder?

File: examples/test/test_openstreetmap.js [hunk 2]
```diff
@@ -28,13 +27,11 @@ describe("OpenStreetMap example", function() {
             address_list: [
                 "Single Street, Suburb",
             ],
-            key: 'testapikey',
```
Comment (hodgestar) [inline examples/test/test_openstreetmap.js]: These changes shouldn't be in your branch -- it looks like develop was merged into this branch badly?

**Pull request closed without merging**